mail/api.py

```diff
@@ -151,6 +152,9 @@ def send_batch(cls, subject, body, recipients,  # pylint: disable=too-many-argum
                 'html': filter_recipient_variables(body),
                 'text': filter_recipient_variables(fallback_text),
                 'recipient-variables': json.dumps(chunk_dict),
+                'X-Mailgun-Variables': {
```

Comment:
@noisecapella can you verify this?

Reply:
I haven't run this code but the `X-Mailgun-Variables` is only meant to be used in an HTTP header, according to this: https://documentation.mailgun.com/en/latest/user_manual.html#attaching-data-to-messages

We are using the mailgun API to send mail, not SMTP, so we need to do this the way the documentation describes for the API

Reply:
changed @noisecapella